DNN Platform/Modules/DnnExportImport/Components/Services/PagesExportService.cs

```diff
@@ -1015,17 +1017,19 @@ private int ImportModulePermissions(ModuleInfo localModule, ExportModule otherMo
                             local.RoleID = roleId.Value;
                         }
 
-                        other.LocalId = localModule.ModulePermissions.Add(local);
-                        var createdBy = Util.GetUserIdByName(_exportImportJob, other.CreatedByUserID, other.CreatedByUserName);
-                        var modifiedBy = Util.GetUserIdByName(_exportImportJob, other.LastModifiedByUserID, other.LastModifiedByUserName);
-                        UpdateModulePermissionChangers(local.ModulePermissionID, createdBy, modifiedBy);
```

Comment:
Why remove these lines?

Reply:
Because the ModulerPermissionId always be -1 until it saved into database, so actually these code do nothing. similar code commented in SaveTabPermissions method as well.